### PR TITLE
RAS-1306: Migrate Party from GCR to GAR

### PR DESCRIPTION
**Pull request by SteveScorfield**
# What and Why?
We are required to migrate across our Docker images from GCR to GAR. This PR migrated party over to GAR.

# How to test?
Build out your env
deploy frontstage using this PR and migrate-frontstage-to-GAR config branch
update the image location in the helm deployment to point to the GAR - currently pointing at the management one to prevent issues with other PRs: europe-west2-docker.pkg.dev/management project id/images

# Jira
https://jira.ons.gov.uk/browse/RAS-1306

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -48,7 +49,7 @@ jobs:
       - name: Setup Google Cloud SDK
         uses: google-github-actions/setup-gcloud@v2
       - run: |
-          gcloud auth configure-docker
+          gcloud auth configure-docker europe-west2-docker.pkg.dev
```
Comment (mfcrocker) [inline .github/workflows/main.yml]:
```suggestion
          gcloud auth configure-docker "$REGISTRY_HOSTNAME"
```

**Review by mfcrocker — APPROVED** [review]
One small improvement

**Review by matthew-robinson-ons — APPROVED** [review]
Deployed and updated yaml. all working and code changes look good.

**Pull request merged**